packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/feature_onboarding/blocs/onboarding_phone_bloc.dart

```diff
@@ -0,0 +1,134 @@
+import 'package:rx_bloc/rx_bloc.dart';
+import 'package:rxdart/rxdart.dart';
+
+import '../../base/extensions/error_model_extensions.dart';
+import '../../base/models/errors/error_model.dart';
+import '../../base/models/user_model.dart';
+import '../models/country_code.dart';
+import '../models/onboarding_phone_errors.dart';
+import '../services/user_service.dart';
+
+part 'onboarding_phone_bloc.rxb.g.dart';
+part 'onboarding_phone_bloc_extensions.dart';
+
+/// A contract class containing all events of the OnboardingBloC
+abstract class OnboardingPhoneBlocEvents {
+  /// Sets the country code for the phone number
+  void setCountryCode(CountryCodeModel countryCode);
+
+  /// Sets the phone number
+  void setPhoneNumber(String phoneNumber);
+
+  /// Checks if the phone number is valid
```

Comment:
Please be specific on the provided code documentation. I guess the `submitPhoneNumber()`event do more than just phone number validation

Reply:
The code itself only submits the phone number to the BE. Along several iterations I forgot to update this comment. Nice catch!